stdlib/file/pom.xml

```diff
@@ -199,7 +184,7 @@
                     </processors>
                     <options>
                         <nativeEntityProviderPackage>
-                            org.ballerinalang.stdlib.file.generated.providers
+                            org.ballerinalang.stdlib.system.generated.providers
```

Comment:
```suggestion
                            org.ballerinalang.stdlib.file.generated.providers
```

Reply:
I think these pom changes are not needed since anyway currently it not working and going to remove soon.

Reply:
Resolved